catalog/core/templates/accounts/profile.html

```diff
@@ -63,9 +63,10 @@
                             humane.log("Profile updated.")
                         },
                         error: function(result) {
-                            console.log(result);
-                            humane.log(["Couldn't update profile: ", result.errors], {timeout: 7500});
-                            console.log(result.errors);
+                            var errobj= JSON.parse(JSON.parse(result.responseText).errors);
```

Comment:
Not sure what this section of code is trying to do, I would like to have better error messages but this appears to just be printing out an email attribute attached to the error object.. Please remove or fix.